paying_for_college/templates/worksheet.html

```diff
@@ -912,7 +912,7 @@ <h3 class="offer-part_heading">
                                         </label>
                                         <p class="aid-form_definition">
                                             Money you borrow from the federal
-                                            government and have to pay back over
+                                            government this year and have to pay back over
```

Comment:
Nitpick that won't hold this PR up:
Keep all line lengths to 80 characters max by joining the lines of the paragraph and then re-add end lines as needed.